UI/volume-control.cpp

```diff
@@ -412,6 +417,39 @@ void VolumeMeter::setInputPeakHoldDuration(qreal v)
 	inputPeakHoldDuration = v;
 }
 
+void VolumeMeter::setPeakMeterType(enum obs_peak_meter_type peakMeterType)
```

Comment:
I don't think you need to specify "enum" before the argument, at least in C++.  Just specifying `obs_peak_meter_type` seems to be sufficient.  If you change this here, please be sure to change it throughout the rest of the C++ code.

Reply:
obs_peak_meter_type is declared in libobs so it is a C-type (as opposed to C++) therefor I cannot use the new C++ style enum-class.

Also obs_peak_meter_type is not a typedef, but a plain enum, so the `enum` is required here. ln libobs code-style it is normal to not make typedefs and use the types like enums and structs directly.

Reply:
VS2017 didn't seem to care when I tried it, compiled it, and ran it.  I'd guess that's because it is in a function definition that is in a C++.  However, I don't know if VS2017 does some work behind the scenes to make these behave equivalently, and I'm admittedly not familiar with all the ins and outs of C.  Do other compilers complain about this?

Reply:
I found some examples of code examples in OBS that shows enums declared/defined in C (not typedefs, plain enums) but called in C++ that do not use the `enum` keyword.
https://github.com/obsproject/obs-studio/blob/189e535014287a7c4999e716c2edb5fc07244b19/libobs/obs.h#L980
https://github.com/obsproject/obs-studio/blob/9a1c27accbb2aaeb07d84d24177944df1fd2995c/UI/window-basic-main.cpp#L650

I haven't found an example showing that you can also drop the `enum` keyword for C enums that are used in C++ function definitions.

Reply:
In C++ you don't need the "enum" or "struct" keywords as all structs/enums are automatically made in to types, but it's not against the rules either (in order to keep interoperability with C).  I myself have occasionally been inconsistent with doing this.  You'll see various places throughout the C++ code where I'll prefix with "struct" or "enum" unnecessarily just because that's what I'm used to doing, so it's not a huge deal in this case.